### ISSUE TITLE
Having an issue during build

### ISSUE THREAD

**Issue by jdrich** [jodit/jodit-react#147]
(Emitted value instead of an instance of Error) Cannot find SourceMap 'data:application/json;base64,"+btoa(unescape(encodeURIComponent(JSON.stringify(a))))+"': Error: Can't resolve './data:application/json;base64,"+btoa(unescape(encodeURIComponent(JSON.stringify(a))))+"' in '\public\js\node_modules\jodit-react\build'

This is in webpack 4 and 5, jodit-react 1.1.1

**Comment by dennisat** [comment]
Same here, I can confirm.

**Comment by DatNguyen-ITC** [comment]
Yep, I got it too, working totally fine on local but it failed during build
